primitives/hp-bridge-dispatch-aggregations/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "hp-bridge-dispatch-aggregations"
```

Comment:
maybe is better to choose a little more generic one: `hp-dispatch` the inner code doesn't know nothing about aggregation (ok the signature of dispatch function trait should know what to send, but we can be generic on it). Sure the destination is not only for bridges.

Reply:
ok